website/cue/reference/components/sources/aws_kinesis_firehose.cue

```diff
@@ -198,8 +198,12 @@ components: sources: aws_kinesis_firehose: {
 					access_key = "secret"
 
 					[transforms.cloudwatch]
-					type = "aws_cloudwatch_logs_subscription_parser"
+					type = "remap"
 					inputs = ["firehose"]
+					drop_on_error = false
+					source = '''
+					. |= parse_aws_cloudwatch_log_subscription_message!(.message)
```

Comment:
Should we put the full `remap` you have in the guide here?

Reply:
Perhaps - though it does make me wonder if the parsing function "does enough" or if it should do more of these things by default.

Reply:
Yeah, that is a good question. It probably could do more. Right now it isn't much more than just JSON parsing with some renames.